server/services/participants.js

```diff
@@ -6,12 +6,81 @@ const { dbClient, collections } = require('../db');
 const { createRows, verifyHeaders } = require('../utils');
 const { userRegionQuery } = require('./user.js');
 
-const getParticipants = async (user) => {
+const setParticipantStatus = async (employerId, participantId, status) => {
+  await dbClient.db[collections.PARTICIPANTS_STATUS].update({
```

Comment:
I've got some concerns about this not being in a transaction. If the first write succeeds and second write fails, we'll end up with no active statuses. I think we could either
1. Wrap these actions in a transaction
2. Use timestamps to mark which status is active (most recent)

Reply:
Good catch, we actually can’t use the 2. approach here due to the limitation of massive.js and also because we would need to perform a scan over the dates to get the most recent one, which could lead to performance issues when doing the join (pretty sure that there's an optimized way to do that using raw query but that requires too much time googling and testing).